packages/quantic/force-app/main/default/lwc/quanticSearchBox/templates/expandableSearchBox.html

```diff
@@ -1,6 +1,6 @@
 <!-- sldsValidatorIgnore -->
 <template>
-  <div class="slds-form-element z-index-high">
+  <div class="slds-form-element z-index-high searchbox__container-min-height">
```

Comment:
I would also call this class simply `searchbox__container` for simplicity and scalability.
imagine in the future we would need to add additional CSS rules to this class, the name will no longer be very relevant.

Reply:
I can't since there is already another element with that class name, I could change it to something more generic however or change the name of that other class `searchbox__container`

Reply:
changed it to `searchbox__container-wrapper` is that better?

Reply:
For now yeah that's better, maybe when refactoring the search box it would be worth rechecking these multiple containers wrapping the search box, maybe they can be simplified.

cause having multiple containers around elements is also something a bit fishy, sometimes it is indeed necessary but it's worth rechecking for a potential simplification.